keras_cv/layers/preprocessing/five_crop.py

```diff
@@ -0,0 +1,105 @@
+# Copyright 2023 The KerasCV Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import tensorflow as tf
+from tensorflow import keras
+
+from keras_cv.layers.preprocessing.base_image_augmentation_layer import (
+    BaseImageAugmentationLayer,
+)
+
+
+@keras.utils.register_keras_serializable(package="keras_cv")
+class FiveCrop(BaseImageAugmentationLayer):
+    """ Crops image into four courners and one center crop.
```

Comment:
Is there a paper or other documentation we can reference somewhere about the value of this preprocessing approach?

If so, it would be great to add a pointer to it in the docstring.